dist/index.html

```diff
@@ -12,9 +12,9 @@ <h1>What's Cookin'?</h1>
       <h2 class="user-info" id="user-info">Welcome, User!</h2>
       <div class="button-container">
         <button type="button" class="home-button" id="home-button">Home</button>
-        <button type="button" class="saved-recipe-button" id="saved-recipe-button">Saved Recipes</button>
-        <button type="button" class="favorite-recipe-button hidden" id="favorite-recipe-button">Save it!</button>
-        <button type="button" class="remove-recipe-button hidden" id="remove-recipe-button">Delete Favorite!</button>
+        <button type="button" class="favorited-recipe-button" id="favorited-recipe-button">Favorite Recipes</button>
+        <button type="button" class="favorite-recipe-button hidden" id="favorite-recipe-button">Save Favorite</button>
```

Comment:
Right now we have "Favorite Recipe" and "Save Favorite" I'm wondering if those names might be confusing to the user. Would naming these "View Favorite Recipes List" and "Favorite Recipe" be any better? Or we could lean into the "recipes to cook" verbiage? I'm wondering what Jenn's thoughts are?

Reply:
I really like the idea of using the "recipes to cook" but wondering if "Add to Recipes to Cook" might be long verbiage for a button. We could say "Add to Favorites" and "View Favorite Recipes".

Reply:
I like that verbiage! I knew that you would have a great idea Jenn, I'll add a ticket for this!